installation_guide/installation_distros.rst: improve readability (#268)
ansible/ansible-documentation: docs/docsite/rst/installation_guide/installation_distros.rst

The ``ansible`` package can always be :ref:`installed from PyPI using pip <intro_installation_guide>` on most systems but it is also packaged and maintained by the community for a variety of Linux distributions.

This document guides you through installing the ``ansible`` package with different package managers.

To add instructions for another distribution to this guide, maintainers for that distribution **must** do the following:

    * Ensure the distribution provides a reasonably up-to-date version of ``ansible``.
    * Ensure that ``ansible-core`` and ``ansible`` versions are kept synchronized as closely as possible, to the extent that the build system allows.
    * Provide a way to contact the distribution maintainers as part of the instructions. Distribution maintainers are also encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ Matrix room.

.. contents::
  :local:

Installing Ansible on Fedora Linux
----------------------------------

To install the minimal ``ansible-core`` package on Fedora run

> 

Reviewer: I commented on another PR but will repeat here - We chose specifically to list ansible package installs first as that is the community's preferred installation. If you want to change this, please open an issue so the community can discuss  the pros/cons of changing the order to put core install first.

Reviewer 2: > I commented on another PR but will repeat here - We chose specifically to list ansible package installs first as that is the community's preferred installation. If you want to change this, please open an issue so the community can discuss the pros/cons of changing the order to put core install first.

I tend to agree.

Reviewer 2: I'll mark this as a draft so we can discuss that point in more detail.

The ``ansible`` package can always be :ref:`installed from PyPI using pip <intro_installation_guide>` on most systems but it is also packaged and maintained by the community for a variety of Linux distributions.

This document guides you through installing the ``ansible`` package with different package managers.

To add instructions for another distribution to this guide, maintainers for that distribution **must** do the following:

    * Ensure the distribution provides a reasonably up-to-date version of ``ansible``.
    * Ensure that ``ansible-core`` and ``ansible`` versions are kept synchronized as closely as possible, to the extent that the build system allows.
    * Provide a way to contact the distribution maintainers as part of the instructions. Distribution maintainers are also encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ Matrix room.

.. contents::
  :local:

Installing Ansible on Fedora Linux
----------------------------------

To install the minimal ``ansible-core`` package on Fedora run

.. code-block:: bash

    $ sudo dnf install ansible-core

> Then, you can install only collections you need using the :ref:`ansible-galaxy<collections_installing>` command.

Reviewer 2: Since this is the Fedora install docs, I'd prefer if we featured the Fedora packaged collections.

Author: @gotmax23 i've removed this line, wfm too. is it ok now?

Reviewer 2: Cool, thanks.

The ``ansible`` package can always be :ref:`installed from PyPI using pip <intro_installation_guide>` on most systems but it is also packaged and maintained by the community for a variety of Linux distributions.

> This document guides you through installing the ``ansible`` package with different package managers.

Reviewer 2: ```suggestion
This document guides you through installing Ansible from different distribution's package repositories.
```

or

```suggestion
This document guides you through installing Ansible from your preferred distribution's package repositories.
```

- "Ansible" instead of " `ansible`", as the instructions include core and collections as well
- "different package managers" isn't entirely correct. The docs are talking about specific distributions. Two distributions may use the same package manager, but that doesn't mean they both have the ansible package.

Author: sounds fair, thanks

> To add instructions for another distribution to this guide, maintainers for that distribution **must** do the following:

Reviewer 2: ```suggestion
To add instructions for another distribution to this guide, package maintainers **must** do the following:
```
This is more concise.

>     * Ensure the distribution provides a reasonably up-to-date version of ``ansible``.

Reviewer 2: I'm not sure if `up-to-date` is supposed to be hyphenated. @oraNod, @samccann, WDYT?

Reviewer: had to google this one.. it's correct with the hyphens in this caste as it's acting as an adjective.

>     * Ensure that ``ansible-core`` and ``ansible`` versions are kept synchronized as closely as possible, to the extent that the build system allows.

Reviewer 2: ```suggestion
    * Ensure that ``ansible-core`` and ``ansible`` versions are kept in sync to the extent that the build system allows.
```

I think this is clearer and less wordy.

The following instructions will guide you through installing the ``ansible`` package with your preferred distribution's package manager.

.. note::

  For maintainers who wish to add distributions to this guide, installation instructions are included here only for distributions with a reasonably up-to-date version of ``ansible``.

  The distribution MUST ensure that ``ansible-core`` and ``ansible`` versions are kept in sync to the extent that the distribution build system allows.

>   Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ Matrix room.

Reviewer 2: Does this really make it more readable?

Author: From my perspective, yes:)

The following instructions will guide you through installing the ``ansible`` package with your preferred distribution's package manager.

> .. note::

Reviewer 3: I don't think an ordered list belongs in a note. We could probably also combine lines 10 and 14 here.

Author: @oraNod i've made it not ordered. Or do you mean something else? If yes, please suggest changes

See the `Fedora Packages index <https://packages.fedoraproject.org/search?query=ansible-collection>`_
for a full list of Ansible collections packaged in Fedora.

> If you prefer to install the batteries included ``ansible`` package run

Reviewer 3: ```suggestion
To install the full ``ansible`` package run
```
